dima/react-movies/src/pages/pageNotFound/pageNotFound.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import ContentContainer from "../../components/contentContainer";
+import Header from "../../components/header";
+import Footer from "../../components/footer";
+import "./pageNotFound.css";
+const PageNotFound = () => {
+  return (
+    <div className="not-found">
+      <div className="first-screen-wrapper first">
+        <ContentContainer>
+          <Header showSearchBtn={false} />
```

Comment:
it is unnecessary to pass  showSearchBtn={false} if you have smthing like `if (showBtn) then` inside of component. showSearchBtn, if you don't pass it,  will be **undefined** and **false** in conditions anyway